src/definitions/power.ts

```diff
@@ -34,6 +34,27 @@ const power = {
     },
     to_anchor: 1000000000,
   },
+  'BTU/s': {
+    name: {
+      singular: 'British thermal unit per second',
+      plural: 'British thermal units per second',
+    },
+    to_anchor: 1055.05585,
+  },
+  'ft-lb/s': {
+    name: {
+      singular: 'Foot-pound per second',
+      plural: 'Foot-pounds per second',
+    },
+    to_anchor: 1.3558,
+  },
+  hp: {
+    name: {
+      singular: 'Horsepower',
+      plural: 'Horsepower',
+    },
+    to_anchor: 745.7,
+  },
```

Comment:
Sounds good. 

Is there any specific criteria that is followed while coming up with these unit names? 

For example, in my use-cases (typically, I follow [Perry's handbook](https://www.accessengineeringlibrary.com/content/book/9780071422949) and [NIST](https://www.nist.gov/pml/special-publication-811/nist-guide-si-appendix-b-conversion-factors/nist-guide-si-appendix-b8)), I haven't come across the usage of `PS`. Perhaps it's not very common? But it's clearly a much concise alternative compared to the longer but more common `horsepower (metric)` or `hp (metric)`.

Meanwhile, I'll add `PS` to the PR.

Reply:
There isn't really any solid criteria, I just noticed that the [Wikipedia's entry for horsepower](https://en.wikipedia.org/wiki/Horsepower#Metric_horsepower_(PS,_cv,_hk,_pk,_ks,_ch)) listed the `PS` abbreviation and referenced it a few times in the section for metric. I recommended it based on that and the fact that I've seen it used before. However, the real reason for the choice is more technical. For a given measure, if any system has a unit with the same name, there is no guarantee which unit would be returned for methods like `getUnit`. It could be the first or the second, no way to know. This is why I recommended `PS`, it's to avoid this short coming in the current implementation. I also didn't like the idea of using `hp (metric)`.

Reply:
Sounds good, no other query.